src/core/types/base/pool.ts

```diff
@@ -351,6 +351,122 @@ export function getAssetsOrder(pool: Pool, assetInfo: AssetInfo) {
 	}
 }
 
+/**
+ * Gets all unique Token values. Adds the liquidity of a Pool together to one stable Liquidity Value. Adds if this is greater than the user given threshold.
+ */
+export function clearPoolsbyLiqThreshold(pools: Array<Pool>, threshold: number, chainId: string) {
+	const out: Array<Pool> = [];
+	let stableCoin: Array<string>;
+	const values: { [tokenaddress: string]: { valuePerToken: number; wheight: number; stableCoin: string } } = {};
+	switch (chainId) {
+		case "phoenix-1":
```

Comment:
yeah I wouldnt use hardcoded ibc denoms here, use this in the config like a Coin type: "amountDenom"
so like "1000000ibc/B3504E092456BA618CC28AC671A71FB08C6CA0FD0BE7C8A5B5A3E2DD933CC9E4"